testdata/exporter/app-slices/layers/other.buildpack.id/new-launch-layer.toml

```diff
@@ -1 +1,2 @@
-launch = true
\ No newline at end of file
+[types]
+  launch = true
```

Comment:
```suggestion
  launch = true

```

Reply:
I know it was already like this, but we really should have a new line at the end of all these data files.

Reply:
Please see https://github.com/buildpacks/lifecycle/pull/537/commits/53c486316a8f616bd11b4c5ecb4a2a304c6b9a93.